examples/append_address_to_webhook.rs

```diff
@@ -1,15 +1,18 @@
-use helius::types::Cluster;
+use helius::error::HeliusError;
+use helius::types::{Cluster, Webhook};
 use helius::Helius;
 
 #[tokio::main]
-async fn main() {
+async fn main() -> Result<(), HeliusError> {
```

Comment:
Very nice! Nit: we have a [`Result` type alias](https://github.com/helius-labs/helius-rust-sdk/blob/6a9a7962250810466bef296ee511578fa3c1940e/src/error.rs#L141-L142) that can be used here to simplify the code from `Result<(), HeliusError>` to `Result<()>`. Can you please update the examples to this?

Reply:
Gotcha!!